src/Plugin.php

```diff
@@ -27,6 +91,6 @@ public static function init() {
 					\WP_CLI::add_command( 'wpml_elasticpress', Command::class );
 				}
 			}
-		} );
+		}, 11 );
```

Comment:
@decodekult, perhaps you can explain why we need priority 11 (a class constant or a comment).

Reply:
+1